src/lib/formats/fsmgr.h

```diff
@@ -158,7 +158,9 @@ class fsblk_t {
 		block_t strong() { return block_t(m_object, false); }
 		block_t weak() { return block_t(m_object, true); }
 
-		const u8 *rodata() { return m_object->rodata(); }
+		u32 size() const { return m_object->size(); }
+
+		const u8 *rodata() const { return m_object->rodata(); }
```

Comment:
Even if technically correct, I don't like hiding the fact that calling rodata() on the underlying iblock_t may be non-const (lazy loading?).  OTOH, maybe const on iblock_t should be on its rodata() and lazy loading, if ever needed, would go under mutable?  Vas, an opinion?